compiler/src/main/kotlin/io/spine/protodata/codegen/CodePrintable.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.protodata.codegen
+
+/**
+ * A piece of code.
+ *
+ * This is an general interface for composable language-specific code-building elements.
+ */
+public interface CodePrintable {
```

Comment:
Why is it `Printable`? Why cannot it be just `Code`? Like `CodeBlock` in Poet, but shorter. I'm not sure such a generic word is a really good idea, but I'm not happy about the `Printable` suffix without a good reason.

We don't print source code on dead pieces of trees for ages. We used to do some 30 years ago. We've got bigger monitors with fast scrolling since then.

Reply:
To print here means to convert into a string.

This is not really `Code` or `CodeBlock`. When I see `Code`, I assume that it is a snippet, one or more formatted, meaningful, and somewhat complete lines. But here, this is more of a building block for code, such as a type name, a method call, a conditional expression, an operator, etc.

Reply:
How about `CodeElement` then?